src/components.test.js

```diff
@@ -0,0 +1,35 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from './redux/configStore';
+import Rockets from './pages/Rockets';
+import Mission from './pages/Mission';
+import Profile from './pages/Profile';
+
+describe('Test UI', () => {
```

Comment:
- [ ]  2. Based on the previous reviewer's request and the requirement you must write tests for components and functions for your project. You did good by adding a snapshot test but you also need to write tests for functions as well.  Kindly add them, please. If you need more information you can slack me :)